src/DynamoCoreWpf/UI/Prompts/DynamoMessageBox.xaml.cs

```diff
@@ -23,6 +26,15 @@ public partial class DynamoMessageBox : INotifyPropertyChanged
 
         #region Public Properties
         
+        [Obsolete("Do not use, instead use CustomDialogResult")]
+        public new bool? DialogResult
```

Comment:
You can just delete, this class is introduced in 2.13.0

Reply:
thanks @QilongTang I'm actually trying to wrap the base class (window) public property to alert clients and devs not to use this property, we can't remove the underlying property, but I want to note that it should not be used.

Are you okay with me no longer setting it at all, that will make my test pass.

Reply:
Sure, this is our own new UI class, we can do whatever we wish is clear

Reply:
okay done, DialogResult is no longer set, but I kept the `new` method around to hide the base class version, and added some notes to the class summary about how to use it.